ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -599,6 +599,7 @@ func (o *AviObjectGraph) BuildHTTPPolicySet(key string, vsNode *nodes.AviEvhVsNo
 
 func (o *AviObjectGraph) BuildHTTPPolicySetHTTPRequestRules(key, httpPSName string, vsNode *nodes.AviEvhVsNode, routeModel RouteModel, filters []*Filter, index int) {
 	requestRule := &models.HTTPRequestRule{Name: &httpPSName, Enable: proto.Bool(true), Index: proto.Int32(int32(index + 1))}
+	vsNode.HttpPolicyRefs[index].RequestRules = []*models.HTTPRequestRule{}
```

Comment:
I did not understand why this change is needed

Reply:
This is to handle update case on filter section of httpRoute